src/components/skeletons/MenuBar.vue

```diff
@@ -83,7 +83,7 @@
                      label="Comparisons"
                      tag="router-link"
                      :active="isActive('judgingBatch')"
```

Comment:
can this be changed as well to ensure consistency?

Reply:
Since scoring related menuList will change in future development, I think there's no need to
This will later be changed to a separate menu rather than a sub-menu of scoring like the existing code @oliverrsebastian